### PR TITLE
Create integration test

### PR DESCRIPTION
**Pull request by h-unterp**

### CODE REVIEW

File: example/test/widget_test.dart [hunk 1]
```diff
@@ -1,30 +1,5 @@
...
-
-    // Tap the '+' icon and trigger a frame.
-    await tester.tap(find.byIcon(Icons.add));
-    await tester.pump();
-
-    // Verify that our counter has incremented.
-    expect(find.text('0'), findsNothing);
-    expect(find.text('1'), findsOneWidget);
-  });
+  testWidgets('Counter increments smoke test', (WidgetTester tester) async {});
```
Comment (eEQK) [inline example/test/widget_test.dart]: if the body is empty, would it be fine to remove this test altogether?

Comment (h-unterp) [reply example/test/widget_test.dart]: The widget test, sure. I made it empty it because I ran it, and it crashed. Needs an integration_test

Comment (ftognetto) [reply example/test/widget_test.dart]: So we should remove this empty test?

Comment (h-unterp) [reply example/test/widget_test.dart]: yes remove empty 😀

**Pull request merged**